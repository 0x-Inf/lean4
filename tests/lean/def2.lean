set_option new_elaborator true

axiom val : nat

definition foo : nat :=
val

noncomputable definition foo : nat :=
val

noncomputable definition bla : nat :=
2
