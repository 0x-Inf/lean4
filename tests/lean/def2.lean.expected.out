def2.lean:5:0: error: definition 'foo' is noncomputable, it depends on 'val'
def2.lean:11:0: error: definition 'bla' was incorrectly marked as noncomputable
